demos/gallery/samples/graph/Les Miserables.js

```diff
@@ -356,10 +353,10 @@ function createData() {
             { "source": 76, "target": 58, "value": 1 }
         ]
     };
-    rawData.nodes.forEach(function (node, idx) {
+    rawData.nodes.forEach(function (node) {
         vertices.push(
             new Vertex()
-                .centroid(idx === 0)
+                .centroid(node.name === "Valjean")
```

Comment:
As a side note: maybe the `centroid` method should belong to Graph and accept a function as its argument?

Reply:
Maybe - centroid v selection seems like they should be more similar.